src/js/checklist.js

```diff
@@ -58,10 +58,44 @@ function populateChecklistFromLocalStorage(checkboxSelector) {
 
 function processChecklist() {
 	var checkboxSelector = '.c-checklist__checkbox input[type="checkbox"]';
-
 	populateChecklistFromLocalStorage(checkboxSelector);
 	processChecklistClick(checkboxSelector);
 }
 
 openLinkedCheckListItem();
 processChecklist();
+
+function registerToggleButton(buttonEl, parentEl) {
+	if (buttonEl && parentEl) {
+		var details = parentEl.querySelectorAll("details");
+		buttonEl.addEventListener("click", function (event) {
```

Comment:
🔍 Can we store this event handler as a named function, so it's easier for us to find later when we need it?

Reply:
yes! Doing it currently! I was going to change the named function of the event listener to changeAttribute. Does that name work? @mxmason

Reply:
Done! Is handleToggleButtonClick a good name for it?